src/CSnakes.CLI/CSnakes.CLI.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <PackAsTool>true</PackAsTool>
+        <ToolCommandName>csnakes</ToolCommandName>
+        <IsPackable>true</IsPackable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="..\CSnakes\CSnakes.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.24324.3" />
+        <PackageReference Include="Superpower" Version="3.0.0" />

```

Comment:
This is required due to how the reference is managed in the `CSnakes` project (and requirements because it's a source generator), it doesn't end up getting added properly to the CLI project, resulting in:

> Unhandled exception: System.IO.FileNotFoundException: Could not load file or assembly 'Superpower, Version=1.0.0.0, Culture=neutral, PublicKeyToken=aec39280ded1b3a7'. The system cannot find the file specified.